docs/notes/todo.md

```diff
@@ -0,0 +1,45 @@
+# Todo
+
+
+## Critical Path Tasks
+
+- [ ] Test and debug Registry routines
+- [ ] Implement file cache
+- [ ] Implement paging
+
+
+## These Tasks Would Be Nice
+
+- [ ] Separate HAL into KM DLL
```

Comment:
If "KM" == kernel-mode, then I can say for sure that our HALs are definitively kernel-mode dlls since a long time now.